staging/src/k8s.io/component-base/metrics/options.go

```diff
@@ -27,11 +27,22 @@ import (
 )
 
 // Options has all parameters needed for exposing metrics from components
+// +k8s:deepcopy-gen=true
 type Options struct {
```

Comment:
so default value for all of those is `nil`? I wonder if this needs to be specified alongside the +optional

Reply:
You mean like `// +kubebuilder:default:=nil`? IIUC that shouldn't be necessary?

Reply:
No, I meant to say "+optional" and mention the default value in a comment